crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -340,10 +339,18 @@ impl Plugin for VisibilityPlugin {
         app.register_type::<VisibilityClass>()
             .configure_sets(
                 PostUpdate,
-                (CalculateBounds, UpdateFrusta, VisibilityPropagate)
+                (UpdateFrusta, VisibilityPropagate)
                     .before(CheckVisibility)
                     .after(TransformSystem::TransformPropagate),
             )
+            .configure_sets(
+                PostUpdate,
+                (CalculateBounds)
+                    .before(CheckVisibility)
+                    .after(TransformSystem::TransformPropagate)
+                    .after(AssetEvents)
+                    .ambiguous_with(CalculateBounds),
```

Comment:
What does it mean to be ambiguous with itself?

Reply:
Ignore all ambiguities between systems in the same set.